core/debugger/engine_debugger.cpp

```diff
@@ -122,6 +122,9 @@ void EngineDebugger::iteration(uint64_t p_frame_ticks, uint64_t p_process_ticks,
 
 void EngineDebugger::initialize(const String &p_uri, bool p_skip_breakpoints, bool p_ignore_error_breaks, const Vector<String> &p_breakpoints, void (*p_allow_focus_steal_fn)()) {
 	register_uri_handler("tcp://", RemoteDebuggerPeerTCP::create); // TCP is the default protocol. Platforms/modules can add more.
+#ifdef UNIX_ENABLED
+	register_uri_handler("unix://", RemoteDebuggerPeerUDS::create);
+#endif
```

Comment:
Currently, only NetSocketUnix implements AF_UNIX. If someone who has access to a Windows machine would like to do it, that would be awesome.

My intention, short-term, is to use it for embedded debugging, as it will reduce the latency if all the messages sent between the processes

Reply:
I'll take a look, implementation should be almost the same as Unix, `AF_UNIX` should be supported since Redstone 4 so it should be fine to use it with a Windows version check.